### ISSUE TITLE
Research bar 

### ISSUE THREAD

**Issue by Paladorn** [CodingTrain/project-trainsite#2]
I doesn't see a research bar on the website. 
I understand that a lot of people directly go on youtube to see your videos, but it could be usefull to add a research bar to the website. It would help people that want to see a specific tutorial video for instance

**Comment by shiffman** [comment]
@nilsweb yes, and I just transferred! Not sure why it doesn't show for you, maybe you have to be "owner"?

**Comment by nilsweb** [comment]
Maybe... It's still in beta so... ¯\\\_(ツ)\_/¯
